nuxt/components/ReciterCard.vue

```diff
@@ -36,7 +36,7 @@ export default class ReciterCard extends Vue {
   private vibrantTextColor: null|string = null;
 
   get image() {
-    return this.avatar || '/defaults/default-reciter-avatar.png';
+    return this.avatar || require('@/assets/img/defaults/default-reciter-avatar.png');
```

Comment:
Should this not be using the `getReciterAvatar` helper function which you had made ?

I  think the same goes for a few other changes you done ?

Reply:
It should ideally, but that would require some refactoring that I don't want to do right now. Leaving it as is for now.